internal/runbits/requirements/install.go

```diff
@@ -0,0 +1,366 @@
+package requirements
```

Comment:
I am starting the refactor of the monolithic `ExecuteRequirementsOperation()` here. The responsibility of this file is simply to handle installing requirements. I envision another file, _uninstall.go_, having the responsibility of uninstalling requirements.